index.html

```diff
@@ -692,15 +692,19 @@
 
         for (var i in consumers) {
           var user = consumers[i].username;
+          var avatar = consumers[i].avatarmail;
           var userdiv = $('<div class="col-sm-6 col-lg-2"></div>');
           var username = $('<p><a href="#" class="btn-flat btn-block btn btn-primary usernamediv" data-code="' + user + '" id="userSelection" role="button">' + $('<div/>').text(user).html() + '</a>')
+          if(!(avatar == '')) {
+            var userava = $('<img src="'+getAvatarURI(avatar)+'" width="16" height="16"/>');
```

Comment:
This will return the gravatar default image for user that have something other than a valid email-address in this field (some do, there is no validation) while not setting any image for those that leave the field blank. That will probably look strange. 

I guess there should be an else-clause adding the default image for users that have not provided a gravatar mail-address.

Reply:
I thought about this but given the current "default" image is done using a displayed character and not an image there's not much I can provide to Gravatar to "gracefully" fall back to the "old" image. That's why I used the "mm" (MysteryMan) image for gravatar if the provided credentials don't check out. Also skimming over the current list of user information it looked mostly valid entries were provided.

I can however display the "mm" gravatar for everybody, not just those that have (invalid) gravatar information set.

These lines currently only affect the gravatar displayed in the menu bar; thus nothing too visible towards other users.

Reply:
Well let's find out if anybody has complains... probably not.